### DISCUSSION TITLE
How to add simple onclick javascript in PHTML?

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#39] [1. Help]
Hi There,

I am adding the button in footer for "Scroll to Top". but I am not able to use javascript/jQuery in the same phtml. 

How can I implement this? Can you please help me on this?

Thanks

**Comment by vovayatsyuk** [comment]
Why can't you use javascript?

**Reply by rutulritz** [reply]
in phtml How can I use over breeze experience?

**Reply by vovayatsyuk** [reply]
https://breezefront.com/integration-tips#migrate-inline-scripts

**Reply by rutulritz** [reply]
Yes Its working

Thank you so much

**Comment by rutulritz** [comment]
I can not find the $('element').stop(); replacement in Cash here : https://github.com/fabiospampinato/cash
please help

**Reply by rutulritz** [reply]
hello @vovayatsyuk ? plesae help

**Reply by vovayatsyuk** [reply]
Are you using it to stop animation? Just remove all jquery animation methods from your soures. If you need an animation, use CSS transition instead.
